apps/src/codebridge/utils/fileUtils.ts

```diff
@@ -60,9 +63,15 @@ export function prepareSourceForLevelbuilderSave(source?: MultiFileSource) {
   if (!source) {
     return {parsedSource: null, validationFile: null};
   }
+
+  // We skip validation and system support files when saving to levelbuilder.
+  // The validation file is split out into a separate field, and system support
+  // files are not part of level start sources.
   const newFiles = Object.fromEntries(
     Object.entries(source.files).filter(
-      ([_, file]) => file.type !== ProjectFileType.VALIDATION
+      ([_, file]) =>
+        file.type !== ProjectFileType.VALIDATION &&
+        file.type !== ProjectFileType.SYSTEM_SUPPORT
```

Comment:
Should we plan to add tests for these file util functions later on?

Reply:
Yeah I'll file a ticket for that

Reply:
unit test ticket: [CT-956](https://codedotorg.atlassian.net/browse/CT-956)